<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TabbedMenuDialog</class>
 <widget class="QDialog" name="TabbedMenuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Camera calibration</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="2">
          <widget class="QLineEdit" name="images_path"/>
         </item>
         <item row="6" column="2">
          <widget class="QLineEdit" name="right_image_name"/>
         </item>
         <item row="8" column="2">
          <widget class="QSpinBox" name="board_heightSpinBox">
           <property name="value">
            <number>6</number>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="image_extension">
           <property name="text">
            <string>.tif</string>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QPushButton" name="load_left">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSpinBox" name="left_camera_index_spinbox">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="9" column="4">
          <widget class="QPushButton" name="calib_cameras">
           <property name="text">
            <string>Calibrate cameras</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Left camera index</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QPushButton" name="browse">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QPushButton" name="load_right">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QSpinBox" name="board_widthSpinBox">
           <property name="value">
            <number>9</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Number of images</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="no_imagesSpinBox">
           <property name="minimumSize">
            <size>
             <width>133</width>
             <height>20</height>
            </size>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Image extension</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Sample images path</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Right image</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Left image</string>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QPushButton" name="real_time_disp">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Real time disparity map</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="left_image_name"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Board height</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Board width</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Right camera index</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="right_camera_index_spinbox">
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Load calibration parameters</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Path</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="PathField">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Load</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Stereo system parameters</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextBrowser" name="textBrowser"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>Set BM state</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Pre filter size</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QSlider" name="preFilterSz">
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="value">
            <number>21</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="pSz">
           <property name="text">
            <string>21</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Pre filter cap</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSlider" name="preFilterCap">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>31</number>
           </property>
           <property name="value">
            <number>30</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="pCap">
           <property name="text">
            <string>30</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_21">
           <property name="text">
            <string>SAD wndow size</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSlider" name="SADWndSz">
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>2</number>
           </property>
           <property name="value">
            <number>21</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="sadSz">
           <property name="text">
            <string>21</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>Texture threshold</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSlider" name="textureth">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="textTh">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Min. disparity</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSlider" name="minDisp">
           <property name="minimum">
            <number>-30</number>
           </property>
           <property name="maximum">
            <number>130</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLineEdit" name="minDisp_2">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>Num.disparities</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSlider" name="numDisp">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
           <property name="singleStep">
            <number>16</number>
           </property>
           <property name="pageStep">
            <number>16</number>
           </property>
           <property name="value">
            <number>6</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::NoTicks</enum>
           </property>
           <property name="tickInterval">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="numDisp_2">
           <property name="text">
            <string>128</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>Uniquness ratio</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QSlider" name="uniq">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>200</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLineEdit" name="uniqRatio">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>Speckle win size</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSlider" name="speckleWndSz">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QLineEdit" name="spkSz">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>Speckle range</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QSlider" name="specklerange">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QLineEdit" name="spkRag">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>378</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="tryBtn">
           <property name="text">
            <string>Save paramters</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Load BM state</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>TabbedMenuDialog</receiver>
   <slot>loadCalibParams()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>665</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calib_cameras</sender>
   <signal>clicked()</signal>
   <receiver>TabbedMenuDialog</receiver>
   <slot>startCalib()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>load_left</sender>
   <signal>clicked()</signal>
   <receiver>TabbedMenuDialog</receiver>
   <slot>loadLeftImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>563</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>686</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>load_right</sender>
   <signal>clicked()</signal>
   <receiver>TabbedMenuDialog</receiver>
   <slot>loadRightImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>688</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browse</sender>
   <signal>clicked()</signal>
   <receiver>TabbedMenuDialog</receiver>
   <slot>browseImagesPath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>real_time_disp</sender>
   <signal>clicked()</signal>
   <receiver>TabbedMenuDialog</receiver>
   <slot>viewDisparityMap()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadCalibParams()</slot>
  <slot>browseImagesPath()</slot>
  <slot>loadLeftImage()</slot>
  <slot>loadRightImage()</slot>
  <slot>startCalib()</slot>
  <slot>viewDisparityMap()</slot>
 </slots>
</ui>
